build.gradle

```diff
@@ -107,30 +107,24 @@ dependencies {
     compile 'org.apache.tika:tika-core:1.17'
     compile 'org.apache.tika:tika-parsers:1.17'
 
+    compile group: 'io.zonky.test', name: 'embedded-postgres', version: '1.2.1'
```

Comment:
can we trust this solution
what is the adventeges of using it?

Reply:
it is fork of previous solution with some fixes and upgrades
https://github.com/zonkyio/embedded-postgres#features